src/providers/bttv/BttvEmotes.cpp

```diff
@@ -58,7 +58,7 @@ namespace {
                 name,
                 ImageSet{Image::fromUrl(getEmoteLinkV3(id, "1x"), 1),
                          Image::fromUrl(getEmoteLinkV3(id, "2x"), 0.5),
-                         Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.25)},
+                         Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.286)},
```

Comment:
Why is this being changed?

Reply:
I believe this was a scaling change from @goldbattle

If you want, I can revert the PR before that commit tomorrow, and then do scaling changes in a separate PR.

Reply:
This comment has details
https://github.com/SevenTV/chatterino7/pull/62#issuecomment-1008244983

I was planning on making a PR upstream to fix and discuss with them, but forgot about it.

Reply:
I am also curious where the numbers come from, why `0.286`?

Reply:
So also @mmattbtw this change is not from this pr? Is is an artifact from a merge?

Reply:
I cherry-picked this commit into this PR. https://github.com/goldbattle/chatterino2/commit/403b947b6351f2c6bc6cf3f756e536cdfc00ab96

Reply:
Ah, i see it is. In future please dont use git merge, rather do a git rebase and then force push the branch.